selenium_tests/lti_emailer/page_objects/add_user_po.py

```diff
@@ -0,0 +1,43 @@
+__author__ = 'hic048'
+
+from selenium.webdriver.common.by import By
+from selenium_tests.lti_emailer.page_objects.emailer_base import EmailerBasePage
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.support.ui import Select
+
+# These are page-specific locators.
+class Locator3(object):
+      _add_people_url = "http://canvas.icommons.harvard.edu/courses/6389/users"
```

Comment:
If we want this to work with any configuration (e.g. in prod, or some other place like a local Canvas) we should use the setting you defined in `SELENIUM_CONFIG['base_url']`.